roms_tools/setup/initial_conditions.py

```diff
@@ -700,44 +703,9 @@ def plot(
             If the field specified by `var_name` is 3D and none of `s`, `eta`, or `xi` are specified.
             If the field specified by `var_name` is 2D and both `eta` and `xi` are specified.
         """
-
-        field = self.ds[var_name].squeeze()
-
-        if len(field.dims) == 3:
-            if not any([s is not None, eta is not None, xi is not None]):
-                raise ValueError(
-                    "Invalid input: For 3D fields, you must specify at least one of the dimensions 's', 'eta', or 'xi'."
-                )
-            if all([s is not None, eta is not None, xi is not None]):
-                raise ValueError(
-                    "Ambiguous input: For 3D fields, specify at most two of 's', 'eta', or 'xi'. Specifying all three is not allowed."
-                )
-
-        if len(field.dims) == 2 and all([eta is not None, xi is not None]):
-            raise ValueError(
-                "Conflicting input: For 2D fields, specify only one dimension, either 'eta' or 'xi', not both."
-            )
-
-        field = self.ds[var_name].squeeze()
-
-        # Get correct mask and horizontal coordinates
-        if all(dim in field.dims for dim in ["eta_rho", "xi_rho"]):
-            loc = "rho"
-        elif all(dim in field.dims for dim in ["eta_rho", "xi_u"]):
-            loc = "u"
-        elif all(dim in field.dims for dim in ["eta_v", "xi_rho"]):
-            loc = "v"
-        else:
-            ValueError("provided field does not have two horizontal dimension")
-
-        mask = self.grid.ds[f"mask_{loc}"]
-        lat_deg = self.grid.ds[f"lat_{loc}"]
-        lon_deg = self.grid.ds[f"lon_{loc}"]
-
-        if self.grid.straddle:
-            lon_deg = xr.where(lon_deg > 180, lon_deg - 360, lon_deg)
-
-        field = field.assign_coords({"lon": lon_deg, "lat": lat_deg})
+        # Check if variable exists
```

Comment:
probably an unnecessary comment with the `if x in y` and the exception message already spelling this out. consider removing.